torch/utils/data/dataloader.py

```diff
@@ -1018,7 +1031,6 @@ def _next_data(self):
                 return self._process_data(data)
 
     def _try_put_index(self):
-        assert self._tasks_outstanding < 2 * self._num_workers
```

Comment:
can you keep the assertion here by replacing `2` with `self._prefetch_factor`?

Reply:
sure!